Project02Eclipse/src/com/hemebiotech/analytics/WriteSymptomDataToFile.java

```diff
@@ -0,0 +1,43 @@
+  package com.hemebiotech.analytics;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+/**
+ * Simple implementation
+ */
+
+public class WriteSymptomDataToFile implements ISymptomWriter {
+
+	private String filePath;
+
+	/**
+	 * @param symptoms a map containing symptoms strings and their quantities.
+	 */
+	public WriteSymptomDataToFile(String filePath) {
+		this.filePath = filePath;
+	}
+
+	/**
+	 * write the symptoms and their quantities in a result.out file.
+	 */
+	@Override
+	public void writeSymptoms(Map<String, Integer> symptoms) {
+
+		// false passed as the 2nd argument of the FileWriter constructor
+		// implies deletion of the contents of the resultat.out file if not empty.
+		try (BufferedWriter writer = new BufferedWriter(new FileWriter(this.filePath, false))) {
```

Comment:
```suggestion
		try (BufferedWriter writer = new BufferedWriter(new FileWriter(this.filePath))) {
```
Il ne sert à rien de préciser le paramètre append car `new FileWriter(filePath)` fait déjà un append false dans l'API.
Pris de la classe `FileWriter`
```
 public FileWriter(String fileName) throws IOException {
        super(new FileOutputStream(fileName));
    }
```
Pris de la classe `FileOutputStream`
```
 public FileOutputStream(String name) throws FileNotFoundException {
        this(name != null ? new File(name) : null, false); // append est false
    }
```